frontend/trpc/routes/documents/internal.ts

```
@@ -0,0 +1,124 @@
+  // Upload PDF and create document
+  upload: companyProcedure
+    .input(z.object({
+      name: z.string(),
+      file: z.string(), // base64 encoded PDF
+      isTemplate: z.boolean().default(false),
+      type: z.number(), // DocumentType enum
+    }))
+    .mutation(async ({ ctx, input }) => {
+      const fileStorageKey = `internal-${Date.now()}-${input.name.replace(/[^a-zA-Z0-9.-]/g, '_')}.pdf`;
+      
+      const [document] = await db.insert(documents).values({
```

ESLint

[error] Replace `.insert(documents)` with `⏎········.insert(documents)⏎········`

(prettier/prettier)